src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2162,26 +2284,61 @@
           short-summary: Name or ID of a storage account to save the packet capture to.
         - name: --storage-path
           short-summary: Fully qualified URI of an existing storage container in which to store the capture file.
-          long-summary: If not specified, the container 'network-watcher-logs' will be
+          long-summary: >
+            If not specified, the container 'network-watcher-logs' will be
             created if it does not exist and the capture file will be stored there.
         - name: --file-path
-          short-summary:
+          short-summary: >
                 Local path on the targeted VM at which to save the packet capture. For Linux VMs, the
                 path must start with /var/captures.
         - name: --vm
           short-summary: Name or ID of the VM to target.
         - name: --filters
           short-summary: JSON encoded list of packet filters. Use `@<file path>` to load from file.
+    examples:
+        - name: Create a packet capture session on a VM
+          text: az network watcher packet-capture create -g MyResourceGroup -n MyPacketCaptureName --vm MyVm --storage-account MyStorageAccount
+        - name: Create a packet capture session on a VM with optional filters for protocols, local IP address and remote IP address ranges and ports
+          text: |
+                az network watcher packet-capture create -g MyResourceGroup -n MyPacketCaptureName --vm MyVm  \\
+                    --storage-account MyStorageAccount --filters "[{\"protocol\":\"TCP\", \\
+                    \"remoteIPAddress\":\"1.1.1.1-255.255.255\",\"localIPAddress\":\"10.0.0.3\", \\
+                    \"remotePort\":\"20\"},{\"protocol\":\"TCP\", \"remoteIPAddress\":\"1.1.1.1-255.255.255\", \\
+                    \"localIPAddress\":\"10.0.0.3\", \"remotePort\":\"80\"},{\"protocol\":\"TCP\", \\
+                    \"remoteIPAddress\":\"1.1.1.1-255.255.255\",\"localIPAddress\":\"10.0.0.3\", \\
+                    \"remotePort\":\"443\"},{\"protocol\":\"UDP\"}]"
+"""
+
+helps['network watcher packet-capture delete'] = """
```

Comment:
These commands need help information in the file.

Reply:
Fixed in my branch.